buildkitd/buildkitd.go

```diff
@@ -50,7 +50,7 @@ func NewClient(ctx context.Context, console conslogging.ConsoleLogger, image, co
 		remoteConsole := console
 		if settings.SatelliteName != "" {
 			remoteConsole = console.WithPrefix("satellite")
-			remoteConsole.Printf("Connecting to %s...", settings.SatelliteName)
+			remoteConsole.Printf("Connecting to %s...", settings.SatelliteDisplayName)
```

Comment:
Does this fix the `connecting to tcp://blah` message we see in CI logs?

Reply:
This makes it so we don't show the ugly hidden name, and instead the pretty name.